source/main/physics/Beam.cpp

```diff
@@ -3459,7 +3459,7 @@ void Actor::ToggleHooks(int group, hook_states mode, int node_number)
                             continue;
 
                         // exclude this truck and its current hooknode from the locking search
-                        if (this == actor && i == it->hk_hook_node->id)
+                        if (this == actor && i == it->hk_hook_node->pos)

```

Comment:
But we can't merge this without updating the parser first, right?

Reply:
I think we should walk feature-by-feature, always checking if the importer works right, then replacing IDs with positions, testing and merging.

Reply:
Sounds perfect.

Reply:
After thinking about it some more I think that `id` was wrong, and `pos` is the way to go here.

Reply:
Same as: https://github.com/RigsOfRods/rigs-of-rods/pull/1840#pullrequestreview-180252993